packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -74,11 +74,11 @@ export class EpochTracker implements IPersistedFileCache {
         // Limits the max number of concurrent requests to 24.
         this.rateLimiter = new RateLimiter(24);
 
-        // We need this for GC testing until we properly plumb through the snapshot expiration policy (see PR #11168)
+        // Matches the logic for the policy defined in odspDocumentStorageServiceBase.ts
         this.snapshotCacheExpiryTimeoutMs =
```

Comment:
This logic is in both epochTracker and odspDocumentStorageServiceBase. Shouldn't this happen in the odspDocumentStorageServiceBase which defines the policy and the epochTracker just uses that policy?

Reply:
It's very hard to do that h less I really missed something. That other PR shows how. I'd rather not go to that length to plumb this TestOverride that were already sneaking around for.

Reply:
I am concerned about the actual policy of 2 days which is read in 2 different places which can result in these diverging. I do see why it's hard though. The EpochTracker is created before the OdspDocumentService.
Actually, the EpochTracker is passed to the OdspDocumentService. So, this can be done in EpochTracker and the OdspDocumentService can read from it. It's all internal so I am not worried about this atm, it can be done after this PR.

Reply:
Ok.  I'll open a follow-up task

Reply:
Can you please add a link to the task here for traking / follow-up purposes.